I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 120, in script
    return
Exception: Could not find return label (u'C:\\pg\\Vitalik\\RenPy\\pr1/game/script.rpy', 1551529886, 46).

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 120, in script
    return
  File "Z:\home\andrey\soft\RenPy\renpy-6.99.14.3-sdk\renpy\ast.py", line 1448, in execute
    next_node(renpy.game.context().lookup_return(pop=True))
  File "Z:\home\andrey\soft\RenPy\renpy-6.99.14.3-sdk\renpy\execution.py", line 629, in lookup_return
    raise Exception("Could not find return label {!r}.".format(self.return_stack[-1]))
Exception: Could not find return label (u'C:\\pg\\Vitalik\\RenPy\\pr1/game/script.rpy', 1551529886, 46).

Windows-7-6.1.7601-SP1
Ren'Py 6.99.14.3.3347
Виталик Алкаштити 1.0
Sun May 26 22:28:22 2019
